frontend/src/pages/org/settings/components/billing.ts

```diff
@@ -195,6 +196,22 @@ export class OrgSettingsBilling extends TailwindElement {
     `;
   }
 
+  private getPlanName(planId: string) {
```

Comment:
Is this worth doing for localization, or just add capitalize() for now?

Reply:
I think this is worth doing, nice to have the hook when/if we implement localization later.